frontend/src/metabase/visualizations/echarts/cartesian/model/dataset.unit.spec.ts

```diff
@@ -309,4 +310,39 @@ describe("dataset transform functions", () => {
       expect(result).toEqual([]);
     });
   });
+
+  describe("getDatasetExtents", () => {
+    const keys = ["series1", "series2"];
+
+    test("should return correct extents for each series", () => {
+      const dataset = [
+        { series1: -100, series2: 4 },
```

Comment:
I think we should add one more series here just to confirm that the function doesn't return extents for series that are not in `keys`